lib/query.js

```diff
@@ -417,7 +422,7 @@ Query.prototype.foreach = function (policy, dataCb, errorCb, endCb) {
  *
  * @param {QueryPolicy} [policy] - The Query Policy to use for this operation.
  *
- * @returns {Promise<Record[]>}
+ * @returns {Promise<object[]>}
```

Comment:
Maybe this is necessary to make the TypeScript type definitions work, but I think when it comes to generating the API docs for the client, just specifying a generic `object` type for the return type is worse.